app/src/main/java/com/zhengcode/detask/activities/dashboard/DashboardChatsActivity.kt

```diff
@@ -0,0 +1,98 @@
+package com.zhengcode.detask.activities.dashboard
+
+import android.content.Intent
+import android.os.Bundle
+import android.util.Log
+import androidx.appcompat.app.AppCompatActivity
+import com.google.firebase.database.*
+import com.xwray.groupie.GroupAdapter
+import com.xwray.groupie.Item
+import com.xwray.groupie.ViewHolder
+import com.zhengcode.detask.R
+import com.zhengcode.detask.activities.messages.ChatLogActivity
+import com.zhengcode.detask.adapters.tasks.TasksAdapter
+import com.zhengcode.detask.models.ChatMessage
+import com.zhengcode.detask.models.OfferedTask
+import com.zhengcode.detask.models.UserWithName
+import com.zhengcode.detask.utils.Helpers
+import com.zhengcode.detask.utils.TaskStatus
+import com.zhengcode.detask.views.DashboardChatRow
+import kotlinx.android.synthetic.main.activity_dashboard_chats.*
+import kotlinx.android.synthetic.main.dashboard_chat_row.view.*
+
+class DashboardChatsActivity : AppCompatActivity() {
+    companion object {
+        val TAG = "DashboardChatsActivity"
+    }
+
+    val adapter = GroupAdapter<ViewHolder>()
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_dashboard_chats)
+
+        recyclerView_dashboard_chats.adapter = adapter
+        // set itemClickListener to your adapter
+        adapter.setOnItemClickListener { item, view ->
+            Log.d(TAG, "123")
+            val intent = Intent(this, ChatLogActivity::class.java)
+            val row = item as DashboardChatRow
+            val chatPartnerUser = row.chatPartnerUser
+
+            // This is a technical debt. Because of the way we implemented the original messaging...
+            val stubTask = OfferedTask(1.0, "", "", 1.0, 1.0, "", "",
```

Comment:
One of the values here should be `chatPartnerUser.username`, given to the username field. I overlooked this. Will fix this in another PR. 

This causes the ChatLogActivity to not have a name of the otherparty